### PR TITLE
[4724] Authorizer service

### PR DESCRIPTION
**Pull request by rhinoandre**

### CODE REVIEW

**Comment by rhinoandre** [comment]
Everything done, just wait for code review

File: src/authorizer/lib/auth-token-decryptor.spec.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import decryptor from './auth-token-decryptor';
+import jwt from 'jsonwebtoken';
+import {Unauthorized} from 'http-errors';
+
+describe('Decryptor', () => {
+  beforeEach(() => {
+
```
Comment (danielgallardo) [inline src/authorizer/lib/auth-token-decryptor.spec.ts]: Empty block. Could this 'beforeEach' be removed?

File: config/authorizer.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 userAuthorizer:
   name: ${self:custom.config.resourcesPrefix}-${self:service}-userAuthorizer
   resultTtlInSeconds: 0
-  arn: arn:aws:lambda:us-east-1:862946620411:function:MoonMail-v3-dev-Auth-userAuthorizer
+  arn:
+    Fn::ImportValue: ${self:custom.config.resourcesPrefix}-userAuthorizer
   identitySource: method.request.header.Authorization
```
Comment (rhinoandre) [inline config/authorizer.yml]: Add a new blank line

File: src/authorizer/lib/auth-token-decryptor.spec.ts (hunk 1 above)
Comment (rhinoandre) [reply src/authorizer/lib/auth-token-decryptor.spec.ts]: It's done!

File: src/authorizer/lib/auth-token-decryptor.spec.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import decryptor from './auth-token-decryptor';
+import jwt from 'jsonwebtoken';
+import {Unauthorized} from 'http-errors';
+
+describe('Decryptor', () => {
+  it('should decrypt the given token using jwt.verify', () => {
+    // AND
```
Comment (rhinoandre) [inline src/authorizer/lib/auth-token-decryptor.spec.ts]: It should be GIVEN, not AND

File: src/authorizer/userAuthorizer/getJsonWebKeys.spec.ts [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+jest.mock('request');
+
+describe('Get JSON web keys', () => {
+  afterEach(() => {
+    request.mockReset();
+  });
+
+  it('should call the API to get the JSON web keys object and cache the result', async () => {
+    // request.mockImplementation((options, callback) => callback(null, {statusCode: 200}))
```
Comment (rhinoandre) [inline src/authorizer/userAuthorizer/getJsonWebKeys.spec.ts]: Remove this comment

**Pull request merged**